homeassistant/components/islamic_prayer_times/coordinator.py

```diff
@@ -21,10 +21,10 @@
 class IslamicPrayerDataUpdateCoordinator(DataUpdateCoordinator[dict[str, datetime]]):
     """Islamic Prayer Client Object."""
 
+    config_entry: ConfigEntry
+
     def __init__(self, hass: HomeAssistant, config_entry: ConfigEntry) -> None:
```

Comment:
We're not using the `config_entry` parameter now.

Reply:
I had to add this because mypy is complaining that `self.config_entry` is None here 
https://github.com/home-assistant/core/blob/d85fddb185e94472f7e372c42c9860bcf7579fba/homeassistant/components/islamic_prayer_times/coordinator.py#L38

Reply:
I'm talking about the parameter on line 26.

Reply:
```suggestion
    def __init__(self, hass: HomeAssistant) -> None:
```